includes/upload/UploadStash.php

```diff
@@ -648,6 +647,7 @@ public function exists() {
 class UploadStashNotAvailableException extends MWException {};
 class UploadStashFileNotFoundException extends MWException {};
 class UploadStashBadPathException extends MWException {};
+class UploadStashFileDoesntExistException extends UploadStashBadPathException {};
 class UploadStashFileException extends MWException {};
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this empty statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1116)